_includes/binary2.html

```diff
@@ -153,21 +153,21 @@
   </div>
 
 <style>
-    #binary			{width: 100%;}
-    #decimal		{font-family: Arial, Helvetica, sans-serif; float: left; font-size: 5vw; width: 21vw; margin: 2.7vw 0 0 2vw; float: left}
+    #binary			{width: 100%; display: flex; margin-top: 10px;}
+    #decimal		{font-family: Arial, Helvetica, sans-serif; font-size: 5vw; width: 21vw; margin: 2.7vw 0 0; text-align:right; margin-left: 10px;}
     .column_heading	{font-size: 1.6vw;}
     .binary			{width: 620px; margin: auto; display: flex}
     #container		{width: 560px; display: block; margin: auto; margin-bottom :30px}
     #too_big		{display: none; font-style: italic; clear: left}
     #operators		{width: 680px; margin: auto; margin-left: -20%}
-    .decimal		{font-size: 48px; width: 160px; font-size:48px; margin: 25px 0 0 10px; float:left}
-    #result			{font-size: 48px; width: 160px; margin: 25px 0 0 10px; float:left}
-    .column			{font-family: Arial, Helvetica, sans-serif; float: left; text-align: center; width: 54px}
+    .decimal		{font-size: 4vw; width: 400px; margin: 30px 0 0 10px; float:left; border-radius: 1.3vw;}
+    #result			{font-family: Arial, Helvetica, sans-serif; float: left; font-size: 4vw; width: 400px; margin: 2.7vw 0 0; float: left; margin-left: 10px;}
```

Comment:
I would like to suggest you one thing, **DO NOT use vw alone with font-size and border-radius.**
### Reason: 
100vw stands for 100% of the width of the display which is visible(vw: viewport width). 
Using it with font size and border-radius will cause huge issues with different screen sizes. That is, if the browser's width is reduced, the font size will also be reduced in equal proportions. So in mobile devices, where width is smaller than desktop, the font would become extremely small.

Check this out: [https://css-tricks.com/fun-viewport-units/](https://css-tricks.com/fun-viewport-units/)
Please give it a good read 🙂